### ISSUE TITLE
ACS - Black Facemask

### ISSUE THREAD

**Issue by xxATOMxx** [Monika-After-Story/MonikaModDev#6443]
So, some time ago i've made a black facemask, and posted it as a reddit spritepack. Now i think that it _may be_ official spritepack worthy, after i'll make some edits to it.
Here are the 2560x1700 sprites for it:
![acs-atomreal_facemask_black2-5](https://user-images.githubusercontent.com/55636014/94916625-f23b8000-04a6-11eb-804c-326a7e21197c.png)
![acs-atomreal_facemask_black2-0](https://user-images.githubusercontent.com/55636014/94916628-f4054380-04a6-11eb-9a0f-eee824e6e1d3.png)
And Here are the 1280x850 sprites for it (like the reddit version, but the edges are now blurred):
![acs-atomreal_facemask_black2-5](https://user-images.githubusercontent.com/55636014/94916745-244ce200-04a7-11eb-8f94-56391bcceaf2.png)
![acs-atomreal_facemask_black2-0](https://user-images.githubusercontent.com/55636014/94916754-26af3c00-04a7-11eb-90b0-99c491e259cc.png)
I'll think of a new name for that facemask later.
I hope someone will tell me how can i improve it.

**Comment by xxATOMxx** [comment]
Forgot to tell, it already has a gift reaction

**Comment by ThePotatoGuy** [comment]
oops, didn't see this, I'll try and get this in

**Comment by ThePotatoGuy** [comment]
Can you upload the gift reaction code here?

**Comment by xxATOMxx** [comment]
yeah, [here it is](https://drive.google.com/drive/folders/1IxwzFI19sWsQX90-FWV7hzIHpWZM7nKF?usp=sharing), the thing is that i'm not very good at writing Monika, and the function i used to wear the facemask is pretty rough
